src/Sylius/Bundle/PayumBundle/Payum/Offline/Action/CapturePaymentAction.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+* This file is part of the Sylius package.
+*
+* (c) Paweł Jędrzejewski
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Sylius\Bundle\PayumBundle\Payum\Offline\Action;
+
+use Payum\Core\Security\TokenInterface;
+use Payum\Offline\Constants;
+use Sylius\Bundle\PayumBundle\Payum\Action\AbstractCapturePaymentAction;
+use Sylius\Component\Payment\Model\PaymentInterface;
+
+/**
+ * @author Antonio Peric <antonio@locastic.com>
+ */
+class CapturePaymentAction extends AbstractCapturePaymentAction
```

Comment:
It is better to reuse generic capture action and implement fillpaymentdetails action.